src/routes/profile-route/profile-fests-route.jsx

```diff
@@ -75,91 +119,87 @@ export const ProfileFestsRoute = () => {
           </div>
           <div className="flex flex-col">
             {!loading &&
-              data.fests.map((items, index) => {
-                const startDate = new Intl.DateTimeFormat("en-US").format(
-                  new Date(items.startDate)
+              data.fests.map((fest) => {
+                const startDate = new Intl.DateTimeFormat("en-GB").format(
+                  new Date(fest.startDate.replace(/-/g, "/"))
+                );
+                const endDate = new Intl.DateTimeFormat("en-GB").format(
+                  new Date(fest.endDate.replace(/-/g, "/"))
                 );
-                const endDate = new Intl.DateTimeFormat("en-US").format(
-                  new Date(items.endDate)
+                const attendeeStatus = fest.attendees.some(
+                  (attendee) => attendee.username === currentUser.username
                 );
-                // const isClicked = items.some((click) => click.id === items.id);
```

Comment:
If this isn't going to be used it should be removed!

Reply:
festDateInFuture is used below in the <Button>

Reply:
Forgive me, I was referring to the commented line but I realize you removed it